src/duties/pr.js

```diff
@@ -30,7 +30,7 @@ let bigPr = ({ danger, warn, config: { prNetChangeThreshold } }) => {
 }
 
 let noPrDescription = ({ danger, fail }) => {
-  if (danger.github.pr.body.length === 0) {
+  if (danger.github.pr.body === null) {
```

Comment:
This seems unrelated, but either way we should probably check both:
https://github.com/artsy/peril-settings/blob/master/org/allPRs.ts#L20